content/en/docs/access-control-and-account-management/_index.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "Accecc Control and Account Management"
+description: "Accecc Control and Account Management"
+layout: "single"
+
+linkTitle: "Accecc Control and Account Management"
+weight: 4500
+
+icon: "/images/docs/docs.svg"
+
+---
+
+## Installing KubeSphere and Kubernetes on Linux
```

Comment:
@wansir @Sherlock113 Add an overview of this chapter in this section, remove the unrelated content,